lua/autorun/server/sv_daily_restart.lua

```diff
@@ -428,7 +428,7 @@ end
 local function getHoursUntilRestartHour()
     local hoursLeft = 24
     local restartHour = DESIRED_RESTART_HOUR
-    local currentHour = tonumber( os.date( "%H" ) )
+    local currentHour = tonumber( os.date( "!%H" ) )
```

Comment:
Could also do `os.date( "!*t" ).hour`. Not much of a performance difference between the 2 though